turbopack/crates/turbopack-tests/tests/snapshot/css/css-modules/output/turbopack_crates_turbopack-tests_tests_snapshot_css_css-modules_input_8f2e7f._.js.map

```diff
@@ -2,7 +2,7 @@
   "version": 3,
   "sources": [],
   "sections": [
-    {"offset": {"line": 6, "column": 0}, "map": {"version":3,"sources":["turbopack://[project]/turbopack/crates/turbopack-tests/tests/snapshot/css/css-modules/input/style.module.css [test] (css module)"],"sourcesContent":["__turbopack_export_value__({\n  \"module-style\": \"style-module__cu3fEW__module-style\",\n});\n"],"names":[],"mappings":"AAAA;AACA;AACA"}},
+    {"offset": {"line": 6, "column": 0}, "map": {"version":3,"sources":["[project]/turbopack/crates/turbopack-tests/tests/snapshot/css/css-modules/input/style.module.css [test] (css module)"],"sourcesContent":["__turbopack_export_value__({\n  \"module-style\": \"style-module__h5N2Da__module-style\",\n});\n"],"names":[],"mappings":"AAAA;AACA;AACA"}},
```

Comment:
Is this intentional that we still have `[project]` in `sources[number]` or did you mean something else in the PR title?

Reply:
`DiskFileSystem` now takes a `UriScheme` used when generating file uris. In snapshot tests, this doesn't use `UriScheme::File`, so that these do not embed the recorder's absolute path into them and they remain portable.

I'm not sure why `turbopack://` was removed here though, `[project]` is intentional.